website_and_docs/content/documentation/webdriver/troubleshooting/logging.en.md

```diff
@@ -27,6 +27,12 @@ take a look at the [Selenium Logger project](https://github.com/titusfortner/sel
 Python logs are typically created per module. You can match all submodules by referencing the top
 level module. So to work with all loggers in selenium module, you can do this:
 {{< gh-codeblock path="/examples/python/tests/troubleshooting/test_logging.py#L5" >}}
+You must also set a handler (`StreamHandler`, `FileHandler('/path/to/log')`, etc).
+For example, to see logs in the console, you can do this:
+```py
+handler = logging.StreamHandler()
+logger.addHandler(handler)
+```
```

Comment:
How about simply referring to https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk//examples/python/tests/troubleshooting/test_logging.py#L9-L10 as a code block?

Reply:
That's an option too. The changes I added show the `StreamHandler` for getting console logging, which I think is a more common use case. I guess it would be good to show both, so users see how to do file and console logging. I'll update the PR to show both.